eventmesh-spi/src/test/java/org/apache/eventmesh/spi/EventMeshExtensionFactoryTest.java

```diff
@@ -17,13 +17,24 @@
 
 package org.apache.eventmesh.spi;
 
+import org.apache.eventmesh.spi.example.TestPrototypeExtension;
+import org.apache.eventmesh.spi.example.TestSingletonExtension;
+import org.junit.Assert;
 import org.junit.Test;
 
 public class EventMeshExtensionFactoryTest {
 
     @Test
-    public void getExtension() {
-        TestExtension extensionA = EventMeshExtensionFactory.getExtension(TestExtension.class, "extensionA");
-        extensionA.hello();
+    public void getSingletonExtensionTest() {
```

Comment:
Hi @ruanwenjun , I think the `testGetSingleExtension` is better name. the prefix `testXXX` is better.

Reply:
Yes, you are right. I will fix tonight.

Reply:
done